shell/assets/styles/themes/_dark.scss

```diff
@@ -50,7 +49,6 @@
   --default-light-bg           : #{rgba($dark, 0.05)};
   --slider-light-bg            : #{rgba($darker, 1)};
   --slider-light-bg-right      : #{rgba($darker, 0)};
-  --primary-keyboard-focus     : #{$keyboard-focus};
```

Comment:
The focus ring ends up being the same color in both light and dark. Just like the `primary-hover-bg` variable, we don't need to specify this again for the dark theme.